include/fiction/utils/layout_utils.hpp

```diff
@@ -350,7 +351,56 @@ Lyt convert_to_fiction_coordinates(const sidb_cell_clk_lyt_siqad& lyt) noexcept
 
     return lyt_new;
 }
+/**
+ * Generates a random cell between two given cells.
+ *
+ * @tparam Lyt Cell-level layout type.
+ * @param cell1 First given cell.
+ * @return cell2 Second given cell.
```

Comment:
"First" and "Second" might not be very descriptive attributes of the passed entities.